Android/PowerUp/src/com/example/powerup/MainActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.example.powerup;
+
+import com.example.powerup.R;
+
+import android.app.ListActivity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+
+public class MainActivity extends ListActivity {
+
+	@Override
+	protected void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		Intent myIntent = new Intent(MainActivity.this, Game.class);
```

Comment:
This is done as Game activity would be called by a button(onClickListener) in the MainActivity. The MainActivity would be converted into a Start Page.
[Already Done: StartPage created in the PowerUpApp Repo]